packages/cardano-services/src/Http/HttpServer.ts

```diff
@@ -122,11 +126,34 @@ export class HttpServer extends RunnableModule {
       HttpServer.sendJSON(res, new ProviderError(ProviderFailure.Unhealthy, err), err.status || 500);
     });
 
-    const buildInfo = JSON.parse(process.env.BUILD_INFO ?? '{}');
-    const deploymentInfo = { ...buildInfo, startupDate: new Date() };
-    const metaHandler = (_: express.Request, res: express.Response) => res.json(deploymentInfo);
-    this.app.get('/meta', metaHandler);
-    this.app.post('/meta', metaHandler);
+    const serverMetadataHandler = async (req: express.Request, res: express.Response) => {
+      this.logger.debug('/meta', { ip: req.ip });
+      let metadata: ServerMetadata;
+      try {
+        const buildInfo: BuildInfo = JSON.parse(this.#config.buildInfo ?? '{}');
+        metadata = { ...buildInfo, startupTime: Date.now() };
+      } catch (error) {
+        this.logger.error(`Invalid JSON format of process.env.BUILD_INFO: ${error}`);
```

Comment:
Better to validate this at startup

Reply:
Applied with 88a41a2a0027e9769d42f40a23bf0f851b81b5da together with your suggestion. @rhyslbw